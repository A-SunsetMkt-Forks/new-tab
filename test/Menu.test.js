'use strict';

const Menu = require('../src/components/Menu.html');

describe('Menu root component', () => {
  it('renders correctly', () => {
    const target = document.createElement('div');
    new Menu({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
