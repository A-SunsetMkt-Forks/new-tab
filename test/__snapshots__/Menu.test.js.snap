// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu root component renders correctly 1`] = `
"<div id=\\"m\\"><div id=\\"i\\">â˜°</div>

  <div id=\\"d\\"><a href=\\"chrome-search://local-ntp/local-ntp.html\\" class=\\"m\\">Open Default Tab</a>
    <a href=\\"chrome://bookmarks/\\" class=\\"m\\">Bookmarks Manager</a>
    <a href=\\"chrome://downloads/\\" class=\\"m\\">Downloads</a>
    <a href=\\"chrome://history/\\" class=\\"m\\">History</a>
    <a href=\\"chrome://settings/passwords\\" class=\\"m\\">Passwords</a>

    <hr>

    <a class=\\"m\\">New Tab Settings</a>
    <a href=\\"https://github.com/MaxMilton/new-tab/issues\\" class=\\"m\\">Submit Bug</a></div></div>"
`;
